### PR TITLE
[Merged by Bors] - Make more noise when the EL is broken

### LINKED ISSUE #3814: Make logs for invalid executon blocks louder
## Description

We sometimes only log debug-level `Batch processing failed` logs for errors like `ParentExecutionPayloadInvalid` when we should log at `WARN` level or higher.

See: https://discord.com/channels/605577013327167508/605577013331361793/1053210521991651348

## Steps to resolve

Modify the error handling in the beacon processor so that these errors (which usually indicate a corrupt EL database) are surfaced in a more obvious way.

### PR DESCRIPTION
**Pull request by michaelsproul**
## Issue Addressed

Closes #3814, replaces #3818.

## Proposed Changes

* Add a WARN log for the case where we are attempting to sync chain segments but can't process them because they're building on an invalid parent. The most common case where we see this is when the execution node database is corrupt, causing sync to stall mysteriously (because we're currently logging the failure only at debug level).
* Additionally I've bumped up the logging for invalid execution payloads to `WARN`. This may result in some duplicate logs as we log errors from the `beacon_chain` and then again from the beacon processor. Invalid payloads and corrupt DBs _should_ be rare enough that this doesn't produce overwhelming log volume.

### CODE REVIEW

File: beacon_node/network/src/beacon_processor/worker/sync_methods.rs [hunk 1]
```diff
@@ -513,6 +513,21 @@ impl<T: BeaconChainTypes> Worker<T> {
                     })
                 }
             }
+            ref err @ BlockError::ParentExecutionPayloadInvalid { ref parent_root } => {
+                warn!(
+                    self.log,
+                    "Failed to sync chain built on invalid parent";
+                    "parent_root" => ?parent_root,
+                    "advice" => "check execution node for database corruption",
```
Comment (pawanjay176) [inline beacon_node/network/src/beacon_processor/worker/sync_methods.rs]: I have seen a restart fix this sometimes on some ELs. Might be good to advice restarting the EL as a preliminary step . Maybe

"try restarting execution node, if problem persists, check execution node for database corruption"

**Review by pawanjay176 — APPROVED** [review]
LGTM. Just a minor suggestion

File: beacon_node/network/src/beacon_processor/worker/sync_methods.rs (hunk 1 above)
Comment (michaelsproul) [reply beacon_node/network/src/beacon_processor/worker/sync_methods.rs]: true! restarting Lighthouse after the EL is fixed can also fix things. I'll suggest both.

**Pull request closed without merging**